src/processor/operator/order_by/top_k.cpp

```diff
@@ -8,6 +9,16 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace processor {
 
+std::string TopKPrintInfo::toString() const {
+    std::string result = "Keys: ";
+    result += binder::ExpressionUtil::toString(keys);
+    result += ", Payloads: ";
```

Comment:
Ditto

Reply:
Is this a duplicate from the comment above?

Reply:
Should be called Expressions: like the payload comment above